tests/unittests/CoreGraphics/CGBitmapContextTests.mm

```diff
@@ -44,24 +44,36 @@
     CGContextRelease(context);
 }
 
-DISABLED_TEST(CGBitmapContext, BitmapInfoAPIs_RGB) {
+TEST(CGBitmapContext, BitmapInfoAPIs_RGB) {
```

Comment:
feels a bit sparse on tests given how much you've just implemented. I'd prefer a lot more test coverage hitting edge conditions and comparing against reference platform to verify correctness.